metaci/build/tasks.py

```diff
@@ -229,3 +264,17 @@ def delete_scratch_org(org_instance_id):
         return f"Deleted org instance #{org.id}"
     else:
         return f"Failed to delete org instance #{ord.id}"
+
+
+def launch_one_off_build_worker(build, lock_id: str):
+    """Immediately launch a one-off-build with env-appropriate autoscaler"""
+    app_name = settings.METACI_LONG_RUNNING_BUILD_APP
+    assert (
+        app_name
+    ), "Need to define METACI_LONG_RUNNING_BUILD_APP to run one-off-builds"
+    autoscaler_class = import_global(settings.METACI_WORKER_AUTOSCALER)
+    autoscaler = autoscaler_class(settings.AUTOSCALERS[app_name])
```

Comment:
```suggestion
    autoscaler = autoscaler_class({"app_name": settings.METACI_LONG_RUNNING_BUILD_APP, "worker_type": "not_used", "queues": [], "max_workers": 0, "worker_reserve": 0})
```

That way there's no need for a new entry in the AUTOSCALERS setting.

But the more I look at this, the more it feels like a one-off build dispatcher is a different type of component from an autoscaler. They both need an app name to target and they both have different implementations between local and Heroku, but aside from an app name to target they don't use any of the same configuration.

Reply:
They are both related to scaling the app up and they will tend to use the same API endpoints and protocols (e.g. Heroku API vs. Boto vs. whatever). 

Also, this will all get deleted when we move to Pipeline API, so perhaps we shouldn't overthink it. It's not a central abstraction.

Reply:
@prescod I'd still like to make the change I proposed here, so that it's not necessary to add a new entry to AUTOSCALERS for the long-running app. It feels wrong to force someone who wants to use it to pick values for 4 settings that are not actually going to be used.

Btw, METACI_LONG_RUNNING_BUILD_APP still needs to be mentioned in docs/configuring.rst and/or the section of the readme about autoscaling, right?

Reply:
@prescod I'll approve this to merge and test on staging once the new settings are documented. Thanks for your patience with all the back and forth.